_data/vnext_faq.yml

```diff
@@ -22,7 +22,7 @@ extenders:
            We intend to move the configuration for V3 to the currently used gateway configuration space e.g. zowe.components.gateway therefore the configuration that was Zuul specific will be removed and the one that is Spring Cloud Gateway will be added to this space. 
 
        - number: 3
-         question: How the client certificate authentication will work?
+         question: How will the client certificate authentication work?
```

Comment:
Try to avoid future tense: How does client certificate authentication work?
or: How does client certificate authentication work in Zowe V3?